### PR TITLE
Update for game version v21.5.0

### PR DESCRIPTION
**Pull request by megalon**
This PR was created by comparing game versions `20.1.0` (before the Alt Costumes update) and `21.5.0` (Garfield Update) decompiled with ILSpy.

I created a local repo and committed the contents of 20.1.0, then added in the contents for 21.5.0 and manually checked the differences to see what changed.

I may have missed something, it's difficult to be sure. I was able to convert Garfield's moveset `char_orb` using this new version.
Before the changes in this PR, NASB_Parser failed to convert his moveset.

### CODE REVIEW

**Review by sc2ad — CHANGES_REQUESTED** [review]
Normally I'm not such a stickler for formatting, but the braces and inconsistent indentation bothers me perhaps more than it should. In addition, the missing accessibility modifiers are probably something you don't intend for-- you probably want them to be public? Instead of default internal.

**Comment by megalon** [comment]
Ok, I think I've fixed the accessibility modifiers and the formatting.
Also fixed the forceZ in the HurtBone class. It seems they are just ignoring it now.

File: NASB_Parser/StateActions/SAConfigHitbox.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ public override void Write(BulkSerializeWriter writer)
             writer.Write(TID);
             writer.Write(1);
             writer.Write(Hitbox);
-            writer.Write(ForceZ0);
```
Comment (sc2ad) [inline NASB_Parser/StateActions/SAConfigHitbox.cs:55]: What changed here, are you sure this is correct?

Comment (megalon) [reply NASB_Parser/StateActions/SAConfigHitbox.cs:55]: Yeah, between the two versions that I compared.
Here is a screenshot from the repo I'm using to compare.
The latest update, 21.5.0, is on the right.

image

Comment (megalon) [reply NASB_Parser/StateActions/SAConfigHitbox.cs:55]: The other change in that file is simply hiding the toggle for that boolean in their "EditDraw" function, so I assume it's an unused value now.
image

**Review by sc2ad — APPROVED** [review]

**Comment by megalon** [comment]
I did another pass on the changes today and noticed a few mistakes in my PR, and fixed with the commits above.
Also, to fix the test case, do I need to simply update the char_apple.txt?

**Pull request merged**